catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/SolrFilterDelegateTest.java

```diff
@@ -473,9 +473,9 @@ public void testXpathOR() {
         String combinedExpectedFilter =
                 "{!xpath}xpath:\"(" + xpath + "[contains(lower-case(.), 'example1')] or " + xpath
                         + "[contains(lower-case(.), 'example2')])\"";
-        String expectedIndex =
-                "{!xpath}(xpath_index:\"" + xpath + "[contains(lower-case(.), 'example1')]\") OR "
-                        + "(xpath_index:\"" + xpath + "[contains(lower-case(.), 'example2')]\")";
+        //        String expectedIndex =
```

Comment:
@vinamartin  I commented out the code because the code that used it was also commented out. I figured I should either comment out both, uncomment both, or delete both. I could delete code like this, if you think that would be better.